packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -935,16 +935,18 @@ export default decorate([
                           placeholder={formatMessage(messages.timeoutUnit)}
                         />
                       </FormGroup>
-                      <FormGroup>
-                        <label htmlFor={state.inputFullIntervalId}>
-                          <strong>{_('fullBackupInterval')}</strong>
-                        </label>{' '}
-                        <Number
-                          id={state.inputFullIntervalId}
-                          onChange={effects.setFullInterval}
-                          value={fullInterval}
-                        />
-                      </FormGroup>
+                      {state.isDelta && (
+                        <FormGroup>
+                          <label htmlFor={state.inputFullIntervalId}>
+                            <strong>{_('fullBackupInterval')}</strong>
+                          </label>{' '}
+                          <Number
+                            id={state.inputFullIntervalId}
+                            onChange={effects.setFullInterval}
+                            value={fullInterval}
+                          />
+                        </FormGroup>
+                      )}
```

Comment:
So if I set a full interval on a delta backup job and then change the backup type to a non-delta type, the `fullInterval` setting would still be saved on the server?

Reply:
Yes, is the point.

Reply:
What do you mean?

Reply:
If the user change the type, we store the advanced settings to not let him re-set the interval if he re-change the type.

Reply:
Ok, I don't think it should, ideally, but it's not a big issue for now so let's keep it like that.